socorro/processor/rules/mozilla.py

```diff
@@ -1245,3 +1245,19 @@ def action(self, raw_crash, dumps, processed_crash, tmpdir, status):
         report_type = self.identify_report_type(processed_crash, status)
 
         processed_crash["report_type"] = report_type
+
+
+class SoftErrorsRule(Rule):
```

Comment:
I looked into the possibility of indexing `soft_errors` in place, since it's already on the `processed_crash.json_dump` and forgoing a processing rule entirely (i.e. by using `source_key` in the `socorro/external/es/super_search_fields.py` spec above), but since the value is an array of objects, the data isn't formatted to display (or be string searched) properly. I.e. in the SuperSearch results, Crash Reports tab, the value for the `soft_errors` column would have been "BAD DATA".

Another reason to have a rule is to normalize when this field is effectively empty (i.e. `null`, `[]` or omitted from the minidump-stackwalker output entirely). Though I need to confirm with the requester what behavior they would like to see here.